src/einsteinpy/symbolic/tensor.py

```diff
@@ -266,9 +266,15 @@ def __init__(
             Raised when arguments syms, variables, functions have data type other than list, tuple or set.
         TypeError
             Raised when argument parent_metric does not belong to MetricTensor class and isn't None.
+        ValueError
+            Raised when argument ``syms`` does not agree with shape of argument ``arr``
 
         """
         super(BaseRelativityTensor, self).__init__(arr=arr, config=config, name=name)
+
+        if len(self.arr.shape) != 0 and self.arr.shape[0] != len(syms):
+            raise ValueError(f"invalid shape of argument arr for syms: {syms}")
```

Comment:
This line is incompatible with `py3.5`. I guess there would be multiple way to do this.

Please check CI logs for more details